### ISSUE TITLE
CThun

### ISSUE THREAD

**Issue by isitLoVe** [Flierr/BigWigs#7]
small eyes are off for like 1-5 secs / no dark glare timer during spell cast / phase 2 initialization working / weaken not working and timers are off after first weaken (how can we know its weakened without combatlog data?)

**Comment by isitLoVe** [comment]
for weakening detection:
capture combatlog before and during weakening - check for weakening in log
if no combatlog entry: change code to something like bug trio or 4hm to count stomach tentacle deaths
also check for death msg from tentacles in stomach

**Comment by isitLoVe** [comment]
no death msg of tentacle in combatlog 
i see no way to implement weakening detection
no weakening detection --> no working bars after 1st weakening

**Comment by Flierr** [comment]
Yeah weaken detection will be problematic. Maybe our first goal to fix the timers in phase one. If we get the phase 1 small eyes and beam correct, then everything works fine untill the first weaken. That's already a nice starting point.

@isitLoVe  Can you obtain a combatlog for cthun and either fix those (p1), or post the log here and i will do it.

**Comment by isitLoVe** [comment]
http://pastebin.com/ViVJQU9g

**Comment by Flierr** [comment]
I changed the small eyes and dark glare in phase 1.

Phase 1 small eyes and dark glare needs testing.
Dark glare has no clear "cast" in the log file, so we have to validate the timer based on observation.

**Comment by isitLoVe** [comment]
i will rewrite the whole module

**Comment by isitLoVe** [comment]
Phase1 looks good
Phase2 needs the following timers:
local timeP2GiantEyeStart = 60     --Giant Eye initial timer for phase 2
local timeP2GiantClawStart = 90    --Giant Claw initial timer for phase 2
local timeP2GiantEye = 65      --Giant Eye timer for phase 2
local timeP2GiantClaw = 35     --Giant Claw timer for phase 2
i cant test those alone on the testserver

For weakening detection use a macro: /script BigWigsCThun:ManualWeakend()
After weakening the time since the last action (Tentacle, Eye, Claw) is calculated then the weakening time (45) is added
eg.: tentacles spawn is in15 sec, weakening is triggered, Timer 15 sec is canceled, a new timer is added 45sec weakend + 15 sec time left = 60sec after weakening is the next tentacle spawn

**Comment by isitLoVe** [comment]
just need the time offset after weakening
currently its scripted like that:
gianteye happens at 00:00
weakening 20 sec later (00:20) - (60 seconds to next eye (01:20) - not working)
weakening takes 45 sec (01:05)
the next giant eye will be after 40 sec (00:20 + 45weakening + 40left (01:45)

this is not correct need to find more testers on testserver cause allways in stomach
